MIGRATION.md

```diff
@@ -0,0 +1,22 @@
+## Migration
+
+### Migrating from v3 to v4
+
+V4 has been dropped support for use through Bower. When using Sass, you will need to add `node_modules` to your `includePath` as well as either:
+- Update your code to use the fully qualified package name, i.e.: `@import "@financial-times/o-brand/main"`, or
+- Also `node_modules/@financial-times` to the Sass include path
```

Comment:
Nice idea to list both these options 👍

Reply:
Plus one this is nice! I think there's a missing word in the last line though? "Also [add]" 
```suggestion
- Update your code to use the fully qualified package name, i.e.: `@import "@financial-times/o-brand/main"`, or
- Also add `node_modules/@financial-times` to the Sass include path
```